solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -188,6 +188,7 @@
 public class SolrCore implements SolrInfoBean, Closeable {
 
   public static final String version = "1.0";
+  public static final String DISABLE_ZK_CONFIG_WATCH = "disable.zk.config.watch";
```

Comment:
I would prefer this be named something like `configSet.core.watch.enable`

Reply:
Would `configSet.core.watch.disable` be okay, or should I change the logic to check for the Boolean and default to false?

Reply:
Yeah, you're right.  The boolean should represent the change from its default state.